src/test/php/scriptlet/unittest/workflow/WrapperTest.class.php

```diff
@@ -597,14 +596,36 @@ public function multipleFileUpload() {
 
     $this->loadFromRequest([
       'attachments' => [
-        'name'     => ["test.txt", "test2.txt"],
-        'type'     => ["text/plain", "text/plain"],
-        'tmp_name' => ["/tmp/phpKLEGHj", "/tmp/phpKLEGew"],
+        'name'     => ['test.txt', 'test2.txt'],
+        'type'     => ['text/plain', 'text/plain'],
+        'tmp_name' => ['/tmp/phpKLEGHj', '/tmp/phpKLEGew'],
         'error'    => [0,0],
         'size'     => [0,0]
       ]
     ]);
     $this->assertInstanceOf('array', $this->wrapper->getValue('attachments'));
-    $this->assertInstanceOf('scriptlet\xml\workflow\FileData', $this->wrapper->getValue('attachments')[0]);
+    $this->assertInstanceOf(
+      'scriptlet\xml\workflow\FileData', $this->wrapper->getValue('attachments')[0]
+    );
+  }
```

Comment:
Please don't use backslash literals inside strings. Either use dotted form or (preferrably) an import and `::class` here:

```php
// At the top of this file
use scriptlet\xml\workflow\FileData;

// Here
$this->assertInstanceOf(FileData::class, $this->wrapper->getValue('attachments')[0]);
```

In this case because you're only asserting on types, you might as well do the following and remove the `$this->assertInstanceOf('array'...)` call while doing so.

```php
$this->assertInstanceOf('scriptlet.xml.workflow.FileData[]', $this->wrapper->getValue('attachments'));
```